cl/search/models.py

```diff
@@ -2476,6 +2478,11 @@ def sort_cites(c):
         return 8
 
 
+# class OpinionMeta(type(AbstractDateTimeModel), type(ResourceType)):
+#     pass
+
+
+# class Opinion(AbstractDateTimeModel, ResourceType, metaclass=OpinionMeta):
 class Opinion(AbstractDateTimeModel):
```

Comment:
This was an attempt to make it clear that our `Opinion` model conforms with eyecite's `ResourceType` type (which is just equal to `Hashable`). However, I couldn't get it working.

If I simply do `class Opinion(AbstractDateTimeModel, ResourceType):`, then I get this error: `TypeError: metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases`.

So, okay, let's set the metaclass explicitly to make it happy. But if I do that (what is commented out), then I get this errror: `TypeError: multiple bases have instance lay-out conflict`.

At this point I couldn't figure out a fix so I just decided to not worry about it. Adding this typing is not necessary for making any of this PR work, but I thought it could be a nice to have.